recipes/jdot/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "jdot" %}
+{% set version = "0.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jdot-{{ version }}.tar.gz
+  sha256: e60bd08559a92246700089a8f1b346885b50c81271c174c553e9a712a5786811
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - jdot=jdot.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - jdot
+  commands:
+    - pip check
+    - jdot --help
```

Comment:
This does not exist yet. :disappointed: https://github.com/saulpw/jdot/issues/10

Maybe this is the pressure cooker to implement it..

Reply:
It does [now](https://github.com/saulpw/jdot/pull/13)!